Motorcycle.js

```diff
@@ -1,6 +1,23 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, seats) {
+	if (seats<=2){
```

Comment:
Here, I'd recommend a more conventional approach by calling Car.call only once and then overriding the value of seats if they are greater than 2.
```
	Car.call(this, make, model, year, color, seats, passengers);
	if (this.seats > 2) {
		this.seats = 2;
	}
```